ff2mpv.js

```diff
@@ -1,5 +1,14 @@
 function ff2mpv(url) {
-    browser.runtime.sendNativeMessage("ff2mpv", { url: url });
+    browser.cookies.getAll({
```

Comment:
Instead of unconditionally sending cookies with every play request, we might want to consider adding a second context menu item, something like `Play with MPV (Cookies)`. Otherwise, there's a potential privacy/correlation concern here: I don't necessarily want sites to know (via my session cookies) that I'm using a non-browser player for their media.

Reply:
Yes, that's a very good objection. I haven't thought about that.
Then there's the question what should happen when pressing the Extension Icon or when pressing Ctrl+Shift+V (Maybe add a second shortcut?). Maybe a default option might be set in some preference menu.

Reply:
> Maybe a default option might be set in some preference menu.

This sounds good to me! I think this overall design would be appropriate:

* Two context menus, regardless of preference setting
* If the user sets "Send cookies with URL" in their preferences, the extension icon and shortcut both switch to sending cookies